app/helpers/application_helper.rb

```diff
@@ -22,4 +22,11 @@ def render_flash_message(flash_value)
     end
   end
 
+  def user_has_membership_application?
+    current_user.try(:membership_applications).try(:any?) && current_user.try(:membership_applications).try(:last).try(:persisted?)
+  end
```

Comment:
In order to test this method in `helper_spec.rb` (BTW, that should be renamed to `application_helper_spec.rb`) I had to get around that fact that rspec and devise provide no easy way to stub out a logged in user, other than for controller tests (which we are not doing).  This is what I had to add to the rspec file:
```
describe 'user_has_membership_application?' do
    it 'returns false if no logged in user' do
      def helper.current_user
        nil
      end
      expect(helper.user_has_membership_application?).to eq false
    end

    it 'returns false if user has no application' do
      def helper.current_user
        FactoryGirl.create(:user)
      end
      expect(helper.user_has_membership_application?).to eq false
    end

    it 'returns true if user has an application' do
      def helper.current_user
        user = FactoryGirl.create(:user)
        app = FactoryGirl.create(:membership_application, user: user)
        user
      end
      expect(helper.user_has_membership_application?).to eq true
    end
end
```
You can see that I had to add an instance method to `helper` in order to stub out `current_user`.

There may be a more elegant way to do this, but this might be a useful technique if we use `current_user` in a helper method (again, I don't recommend having this method (and the one below) in a helper method due to excessive DB access and execution of model logic).